src/integration-test/tests/citizen/claimantResponse/pages/claimant-sign-settlement-agreement.ts

```diff
@@ -0,0 +1,17 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  checkboxConfirm: 'input[id=signedTrue]',
+  submit: 'input[id=saveAndContinue]'
```

Comment:
see above comments for defining buttons for pageobjects outside of fields

Reply:
Done